src/Raven.Studio/typescript/viewmodels/database/indexes/indexPerformance.ts

```diff
@@ -1057,16 +1057,7 @@ class indexPerformance extends viewModelBase {
                     this.hitTest.registerTrackItem(currentX, yStart, dx, indexPerformance.trackHeight, op);
                 }
        
-                if (dx >= 5 && op.Name.startsWith("Collection_")) {
-                    context.fillStyle = this.colors.stripeTextColor;
-                    const text = op.Name.substr("Collection_".length);
-                    const textWidth = context.measureText(text).width;
-                    const truncatedText = graphHelper.truncText(text, textWidth, dx - 4);
-                    if (truncatedText) {
-                        context.font = "12px Lato";
-                        context.fillText(truncatedText, currentX + 2, yStart + 13, dx - 4);
-                    }
-                } else if ((op.Name === "Map" || op.Name === "Reduce") && dx >= 6) {
+                if (dx > 30) {
```

Comment:
there was logic to trunc leading `Collection_`. Do we still need it?

Reply:
yes - there was such logic - and I was debating w/ myself whether to keep it or do w/o it 
so IMHO - now that there is much more text to read on the stripes - it is better to show the `collection_` prefix 
since you immediately know this is the collection name and you don't get confused with all other text

Reply:
but then we end up with common prefix and if we don't enough space we have:
collec...., and collection_....

So maybe let's at least do:
companies (Collection), orders (collection) ?

To make prefix more distinct ?

Reply:
![image](https://user-images.githubusercontent.com/17376974/143002145-39357950-98cb-4a16-a145-0a2e41806d8f.png)